src/modules/gauging-stations/helpers.js

```diff
@@ -44,7 +44,7 @@ const abstractionPeriodInObjectParser = inputObject => {
     // For each item in the abstractionPeriod subobject...
     for (const [key, value] of Object.entries(inputObject.abstractionPeriod)) {
       // Take the first letter of the item's label, and make it uppercase. Then bung 'abstractionPeriod' in front of it
-      replicatedObject[`abstractionPeriod${startCase(key[0])}${key.slice(1)}`] = value
+      replicatedObject[`abstractionPeriod${(key[0]).toUpperCase}${key.slice(1)}`] = value
```

Comment:
Looking deeper at the code now, I can't understand why the previous developers split the string up in the first place instead of giving startCase the full string. It would have achieved the same result they got. Not sure if there is something Im missing just seems weird. Making the changes now